client/src/containers/overview/table/view/overview/index.tsx

```diff
@@ -107,6 +109,11 @@ export function OverviewTable() {
 
   return (
     <>
+      <ProjectDetails
+        open={openDetails}
+        setIsOpen={setOpenDetails}
```

Comment:
Instead of passing props, I would suggesting using a `jotai` atom here so the state is easily accessible from anywhere.